lib/fastlane/actions/update_url_schemes.rb

```diff
@@ -0,0 +1,68 @@
+require 'plist'
+
+module Fastlane
+  module Actions
+    class UpdateUrlSchemesAction < Action
+      def self.run(params)
+        path = params[:path]
+        url_schemes = params[:url_schemes]
+
+        hash = Plist.parse_xml(path)
+        hash['CFBundleURLTypes'].first['CFBundleURLSchemes'] = url_schemes
+        File.write(path, hash.to_plist)
+      end
+
+      def self.description
+        'Updates the URL schemes in the given Info.plist'
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(
+            key: :path,
+            env_name: 'FL_UPDATE_URL_SCHEMES_PATH',
+            description: 'The Plist file\'s path',
+            is_string: true,
+            verify_block: verify_path
+          ),
+
+          FastlaneCore::ConfigItem.new(
+            key: :url_schemes,
+            env_name: "FL_UPDATE_URL_SCHEMES_URL_SCHEMES",
+            description: 'The new URL schemes',
+            is_string: false,
+            verify_block: verify_url_schemes
+          )
+        ]
+      end
+
+      def self.output
+        []
+      end
+
+      def self.authors
+        ['kmikael']
+      end
+
+      def self.is_supported?(platform)
+        [:ios, :mac].include? platform
+      end
+
+      def self.verify_path
```

Comment:
I don't understand what you mean here. Is there a way to create actions without an extra class? What's the `options` method?